style.css

```diff
@@ -511,3 +519,245 @@ input:valid {
+    width: 97%;
+    height: 344px;
+    background-image: none;
+  }
+
+  .background-container {
+    height: 100vh;
+    width: 100%;
+    background-image: url(./ImageP1/Header-llustration-desktop-1.png);
+    background-size: 100% 100%;
+    background-repeat: no-repeat;
+    margin-top: -75px;
+    margin-left: 27px;
+    margin-bottom: 122px;
+  }
+
+  .myheader {
+    width: 70%;
+    height: 114px;
+    margin: 0 auto;
+  }
+
+  .break {
+    display: none;
+  }
+
+  p {
+    width: 70%;
+    height: fit-content;
+    font-size: 18px;
+    margin: 24px auto;
+    margin-top: 0;
+  }
+
+  .desktop {
+    display: block;
+  }
+
+  .flex-icons ul {
+    width: 200px;
+    height: 40px;
+    margin: 0 auto;
+    margin-bottom: 45px;
+    padding-left: 0;
+  }
+
+  .flex-icons li {
+    padding: 0 auto;
+  }
+
+  .chevron {
+    margin: 0 auto;
+    margin-top: 0;
+    margin-bottom: 495px;
+  }
+
+  .works {
+    margin-top: 0;
+  }
+
+  header .work {
+    width: 447px;
+    height: 48px;
+    margin-left: 497px;
+    margin-right: 496px;
+    margin-bottom: 52px;
+  }
+
+  .indicator {
+    display: none;
+  }
+
+  .cards {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+    gap: 0;
+    margin-left: 135px;
+    padding-right: 0;
+    width: 84%;
+    height: 966px;
+    margin-bottom: 160px;
+  }
+
+  .card {
+    flex: 1;
+    margin-left: 0;
+    max-width: 327px;
+  }
+
+  .card-img {
+    max-width: auto;
+  }
+
+  .resume h2 {
+    width: 447px;
+    height: 48px;
+    margin: 0 auto;
+  }
+
+  .resume p {
+    width: fit-content;
+  }
+
+  .side-background {
+    background-image: url(./ImageP1/desk-right-background-resized.png);
+    background-repeat: no-repeat;
+    background-position: 100% 20%;
+  }
+
+  .resume {
+    background-image: url(./ImageP1/bottom-left-desk-background-resized.png);
+    background-position-y: 50%;
+  }
+
+  .grid {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    column-gap: 12px;
+    height: 355px;
+    width: 78%;
+    padding: 0;
+    margin-left: 136px;
+  }
+
+  .card-works {
+    margin-top: 0;
+    margin-right: 0;
+    padding-right: 0;
+    padding-left: 0;
+  }
+
+  .user-form {
+    display: grid;
+    grid-template-columns: auto auto;
+  }
+
+  .user-form > p {
+    width: 463px;
+    height: 256px;
+    margin-left: 154px;
+    font-size: 30px;
+    display: flex;
+    align-items: center;
+  }
+
+  form {
+    width: 100%;
+    height: 450px;
+    margin-top: 40px;
+    margin-left: 0;
+    padding-right: 12px;
+    background-image: url(./ImageP1/bottom-right-background-resized.png);
+    background-repeat: no-repeat;
+    background-position-y: bottom;
+  }
+
+  .input-text {
+    width: 82%;
+    height: 48px;
+    padding-right: 0;
+    text-align: left;
+  }
+
+  .text-area {
+    width: 82%;
+    height: 114px;
+    text-align: left;
+  }
+
+  .button {
+    padding-left: 0;
+    margin-right: 460px;
+  }
+
```

Comment:
- [ ] The Form button should be positioned at the left side of the form and not close to the center of the form based on the design template. Kindly position the button properly. I would advise you to use CSS flexbox for the layout of elements in the form section. 👍 😎

### Your Design 

![image](https://user-images.githubusercontent.com/77106962/228955208-ec8880d0-bcfe-4ff8-89fd-600f1a656511.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/228955343-fd17f239-9e93-47f3-9a72-024a8a5881cc.png)